uv.lock

```diff
@@ -0,0 +1,4316 @@
+version = 1
```

Comment:
Question: given that if you are gonna test an instrumentation you need to install some libraries to instrument shouldn't we just ignore the uv.lock file since it will go out of sync with the one we have here?